README.md

```diff
@@ -293,6 +293,28 @@ or an empty string. Conditional votes are a comment representing a vote.
 In order to perform an action based on resolution of one of more conditional votes,
 a player must post a comment describing the resolution before performing said action.
 
+**345** *Dependency Hell*
+
+The owner of a PR may declare one dependency of this PR (henceforth "child") on another
+PR (henceforth "parent"), by including the text
+"This PR is dependent on #<number of parent>" in an unedited post. Any votes above
+this post are voided. A child PR must be of a branch that branches off the
+branch of the parent.
+
+A ":+1:" vote on a child implies a ":+1:" on its parent. This vote shall be read
+as a comment on the parent just before merging for the purposes of other rules.
```

Comment:
Hmm, maybe keep this generic: A vote in favour

After all the ":+1:" format could change ;)